Goal: Fix a syntax error in smb.conf(5)

Fixes: #512843

Status wrt upstream: Reported

Note: Should rather be fixed in the original XML file

Index: samba-3.3.0rc2/docs/manpages/smb.conf.5
===================================================================
--- samba-3.3.0rc2.orig/docs/manpages/smb.conf.5
+++ samba-3.3.0rc2/docs/manpages/smb.conf.5
@@ -7769,7 +7769,7 @@
 This option is only available if Samba was configured with the argument
 \-\-with\-sys\-quotas
 or on linux when
-\./configure \-\-with\-quotas
+ \./configure \-\-with\-quotas
 was used and a working quota api was found in the system\. Most packages are configured with these options already\.
 .sp
 This parameter should specify the path to a script that can set quota for the specified arguments\.
